### ISSUE TITLE
Training setup is different

### ISSUE THREAD

**Issue by resurgo97** [liuzechun/ReActNet#15]
The released codes seem to have different training setup compared to that in the paper. 
(more epochs, larger batch size, label smoothing...)
Did you get higher top-1 accuracy on ImageNet dataset?
Then could you share the result as well please?

**Comment by Ironteen** [comment]
In this paper, the batch size is 256, which means there are 5000 iterations in one epoch, so the 600k iterations equals 120 epoches. Following the setup in the paper, batch_size = 256, 120 epoch， I get 65%+ top-1 accuracy of ResNet-18 in 1-step, which seems to have a certain gap with the results of the paper.

**Comment by resurgo97** [comment]
@Ironteen 
Thanks!
Actually what I meant is, they seem to use label smoothing, lighting augmentation and more epochs in the github code, which is inconsistent with the paper. I wonder if there is any improvement using those tricks.
Of course I can run the given code and see the result but it takes too long.   
Your answer was also very helpful for me though.

**Comment by NicoNico6** [comment]
> @Ironteen Thanks! Actually what I meant is, they seem to use label smoothing, lighting augmentation and more epochs in the github code, which is inconsistent with the paper. I wonder if there is any improvement using those tricks. Of course I can run the given code and see the result but it takes too long. Your answer was also very helpful for me though.

Hi, did you get some conclusions about these tricks?  Does the model benefit from each other?

**Comment by liuzechun** [comment]
Thanks for your interest in our work. We updated the code to correct it. Longer training did not improve the accuracy, for which we use a larger initial learning rate for training 128 epochs per step. Label smoothing might be a default setting in training and I will try running without Lighting augment to see the performance, which I think will not make a big difference.

**Comment by NicoNico6** [comment]
> Thanks for your interest in our work. We updated the code to correct it. Longer training did not improve the accuracy, for which we use a larger initial learning rate for training 128 epochs per step. Label smoothing might be a default setting in training and I will try running without Lighting augment to see the performance, which I think will not make a big difference.

Thanks for your reply! May I also ask have you tested the ReActNet-MobileNet architecture on CIFAR100? I used the provided code and only changed the stride in the first convolution from 2 to 1. But the evaluation result on CIFAR100 is not very strong,  65% Top-1 accuracy. I used standard data argumentation.
